1907-06-17.xml

```diff
@@ -0,0 +1,2658 @@
+                        <cell>6000 Tons</cell>
+                        <cell>15 June</cell>
+                    </row>
+                    <row>
+                        <cell>Prinz Heinrich</cell>
+                        <cell>7000 Tons</cell>
+                        <cell>19 June</cell>
+                    </row>
+                    <row>
+                        <cell>Scharnhorst</cell>
+                        <cell>8000 Tons</cell>
+                        <cell>13 July</cell>
+                    </row>
+                    <row>
+                        <cell>Kieist</cell>
+                        <cell>9000 Tons</cell>
+                        <cell>17 July</cell>
+                    </row>
+                    <row>
+                        <cell>Roon</cell>
+                        <cell>8000 Tons</cell>
+                        <cell>27 July</cell>
+                    </row>
+                    <row>
+                        <cell>Gneisenan</cell>
+                        <cell>8100 Tons</cell>
+                        <cell>10 Aug</cell>
+                    </row>
+                    <row>
+                        <cell cols="3">OUTWARD: for CHINA and JAPAN via SUEZ, ADEN, COLOMBO, PENANG,
+                            SINGAPORE.</cell>
+                    </row>
+                    <row>
+                        <cell><name>Prinz Ludwig</name></cell>
+                        <cell><measure unit="ton" quantity="8965">9700</measure> Tons</cell>
+                        <cell>about <date when="1907-06-10">10 June</date></cell>
+                    </row>
+                    <row>
+                        <cell><name>Zienten</name></cell>
+                        <cell><measure unit="ton" quantity="5295">8100</measure> Tons</cell>
+                        <cell>about <date when="1907-06-26">26 June</date></cell>
+                    </row>
+                    <row>
+                        <cell cols="3">For AUSTRALIA via SUEZ, ADEN, COLOMBO.</cell>
+                    </row>
+                    <row>
+                        <cell><name>Yorek</name></cell>
+                        <cell><measure unit="ton" quantity="7943">9600</measure> Tons</cell>
+                        <cell>about <date when="1907-06-28">28 June</date></cell>
+                    </row>
+                </table>
+                <p>ROYAL ROUMANIAN EXPRESS STEAMER SERVICE</p>
+                <p>ALEXANDRIA TO SMYRNA, MITYLENE, CONSTANTINOPLE AND CONSTANZA - departures from
+                    Alexandria Friday's at 4 p.m. the 7 and 21 June, 5 and 19 July, 2 August, with
+                    the new twin screw S.S. Regele Carol I. Imperatul Trajan, Romania and Dacia</p>
+                <p>FOR FURTHER PARTICULARS APPLY TO THE AGENTS OF THE</p>
+                <p>NORDDEUTSCHER LLOYD at Cairo, Alexandria, Port-Said and Suez.</p>
+                <p>OTTO STERZING, Agent In Cairo, Opera Square.</p>
+                <p>C. H. 
```

Comment:
there is a template for this: https://dig-eg-gaz.github.io/templates/#egyptian-gazette-subscriptions-and-advertisements